### DISCUSSION TITLE
Training a bent text detection model with polygon annotation

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1721] [General]
Hi, if I want to do bend text detection with indeterminate length polygon annotation, does mmocr have support? Where exactly do I need to modify?
For example：
img1：{"points": [[80, 26], [79, 27], [72, 27], [71, 28], [66, 28], [65, 29], [62, 29], [61, 30], [58, 30], [57, 31], [55, 31], [54, 32], [52, 32], [51, 33], [49, 33], [48, 34], [47, 34], [46, 35], [45, 35], [44, 36], [43, 36], [42, 37], [41, 37], [40, 38]]}
img2：{"points": [[122, 59], [123, 59], [124, 60], [126, 60], [128, 62], [129, 62], [130, 63], [131, 63], [132, 64], [133, 64], [145, 76], [145, 77], [146, 78], [146, 79], [147, 80], [147, 82], [148, 83], [148, 87], [149, 88], [149, 92], [148, 93], [148, 96], [147, 97], [147, 99], [146, 100], [146, 101], [145, 102], [145, 103], [142, 106], [142, 107], [136, 113], [135, 113]]}

**Comment by gaotongxiao** [comment]
Sure, as long as you prepare your annotation in the new format: 

https://mmocr.readthedocs.io/en/dev-1.x/migration/dataset.html#introduction-of-the-new-format

And fill up the "polygon" field in the format of [x1, y1, x2, y2,...].

**Reply by htaoruan** [reply]
Thank you very much for your advice, I'll look into it!
